coverage/src/BamViewChart.js

```diff
@@ -240,25 +240,43 @@ function createBamView(bamHeader, data, element, bamViewControlsElement) {
                         .attr('transform', `translate(${margin.left}, ${mainHeight + margin.top + 20})`)
                         .attr('clip-path', 'url(#clip)');
 
+            const barWidth = 1; // Fixed bar width with 1 pixel
+            const numBins = innerWidth; // Number of bins based on inner width
+
+            // Aggregate data into bins
+            const aggregatedData = new Array(numBins).fill(0).map((_, i) => {
```

Comment:
This looks very similar to the `aggregateDataIntoBins` function. Are they the same? Can it be combined?

Reply:
Yes, After rethinking the code structure, I optimized the code. Thanks, Anders, for always pushing me to rethink the code quality.

Reply:
It's looking great